### ISSUE TITLE
Crashing using labeled lists

### ISSUE THREAD

**Issue by madmas** [asciidocfx/AsciidocFX#183]
Hi,
I managed to reproduce crashed of AsciiDocFx by using labeled lists.
To reproduce it, just start a new document and start typing, for example
```
= Document

Sample Test in a line

LabeledList::
```
By the time I hit the second : , it crashes.
Environment: Java 8 on MacOSX, Git rev a8e7800 (master)
As I ran it from the commandline I got the following output:
```
➜  AsciidocFX git:(master) target/appassembler/bin/asciidocfx.sh
   #                 #   #      #                #####  #   #
  # #    ####   ###          ####   ###    ###   #       # #
 #   #  ###    #     #   #  #   #  #   #  #      ####     #
 #####    ###  #     #   #  #   #  #   #  #      #       # #
 #   #  ####    ###  #   #   ####   ###    ###   #      #   #
 ============================================================
 :: Asciidoc FX ::     (v1.4.4)

 Website: http://asciidocfx.com

[Fatal Error] entities.ent:16:3: The markup in the document preceding the root element must be well-formed.
[Fatal Error] l10n.dtd:1:3: The markup in the document preceding the root element must be well-formed.
[Fatal Error] targetdatabase.dtd:4:3: The markup in the document preceding the root element must be well-formed.
2015-12-16 23:06:21.425 java[32264:1292115] [JRSAppKitAWT markAppIsDaemon]: Process manager already initialized: can't fully enable headless mode.
2015-12-16 23:06:21.426 java[32264:1292115] *** -[__NSArrayM insertObject:atIndex:]: object cannot be nil
2015-12-16 23:06:21.428 java[32264:1292115] (
	0   CoreFoundation                      0x00007fff9557703c __exceptionPreprocess + 172
	1   libobjc.A.dylib                     0x00007fff9496176e objc_exception_throw + 43
	2   CoreFoundation                      0x00007fff95422329 -[__NSArrayM insertObject:atIndex:] + 1033
	3   AppKit                              0x00007fff9c050d12 -[NSMenu insertItem:atIndex:] + 487
	4   libawt_lwawt.dylib                  0x0000000138173537 addMenuItem + 185
	5   libawt_lwawt.dylib                  0x0000000138172ea4 -[ApplicationDelegate _updatePreferencesMenu:enabled:] + 211
	6   libawt_lwawt.dylib                  0x00000001381730c9 __Java_com_apple_eawt__1AppMenuBarHandler_nativeSetMenuState_block_invoke_1 + 159
	7   JavaNativeFoundation                0x0000000126442d3a +[JNFRunLoop _performCopiedBlock:] + 17
	8   Foundation                          0x00007fff99a6fd00 __NSThreadPerformPerform + 293
	9   CoreFoundation                      0x00007fff95493a01 __CFRUNLOOP_IS_CALLING_OUT_TO_A_SOURCE0_PERFORM_FUNCTION__ + 17
	10  CoreFoundation                      0x00007fff95485c5c __CFRunLoopDoSources0 + 476
	11  CoreFoundation                      0x00007fff954851bf __CFRunLoopRun + 927
	12  CoreFoundation                      0x00007fff95484bd8 CFRunLoopRunSpecific + 296
	13  HIToolbox                           0x00007fff98c4656f RunCurrentEventLoopInMode + 235
	14  HIToolbox                           0x00007fff98c462ea ReceiveNextEventCommon + 431
	15  HIToolbox                           0x00007fff98c4612b _BlockUntilNextEventMatchingListInModeWithFilter + 71
	16  AppKit                              0x00007fff9c0608ab _DPSNextEvent + 978
	17  AppKit                              0x00007fff9c05fe58 -[NSApplication nextEventMatchingMask:untilDate:inMode:dequeue:] + 346
	18  AppKit                              0x00007fff9c055af3 -[NSApplication run] + 594
	19  libglass.dylib                      0x0000000128dea5ec -[GlassApplication runLoop:] + 1836
	20  Foundation                          0x00007fff99a6fd00 __NSThreadPerformPerform + 293
	21  CoreFoundation                      0x00007fff95493a01 __CFRUNLOOP_IS_CALLING_OUT_TO_A_SOURCE0_PERFORM_FUNCTION__ + 17
	22  CoreFoundation                      0x00007fff95485b8d __CFRunLoopDoSources0 + 269
	23  CoreFoundation                      0x00007fff954851bf __CFRunLoopRun + 927
	24  CoreFoundation                      0x00007fff95484bd8 CFRunLoopRunSpecific + 296
	25  java                                0x000000010529b463 CreateExecutionEnvironment + 871
	26  java                                0x00000001052971ac JLI_Launch + 1952
	27  java                                0x000000010529d4c0 main + 101
	28  java                                0x0000000105296a04 start + 52
	29  ???                                 0x000000000000000c 0x0 + 12
)
2015-12-16 23:06:21.429 java[32264:1292115] *** -[__NSArrayM insertObject:atIndex:]: object cannot be nil
2015-12-16 23:06:21.430 java[32264:1292115] (
	0   CoreFoundation                      0x00007fff9557703c __exceptionPreprocess + 172
	1   libobjc.A.dylib                     0x00007fff9496176e objc_exception_throw + 43
	2   CoreFoundation                      0x00007fff95422329 -[__NSArrayM insertObject:atIndex:] + 1033
	3   AppKit                              0x00007fff9c050d12 -[NSMenu insertItem:atIndex:] + 487
	4   libawt_lwawt.dylib                  0x0000000138173537 addMenuItem + 185
	5   libawt_lwawt.dylib                  0x0000000138172ea4 -[ApplicationDelegate _updatePreferencesMenu:enabled:] + 211
	6   libawt_lwawt.dylib                  0x00000001381730c9 __Java_com_apple_eawt__1AppMenuBarHandler_nativeSetMenuState_block_invoke_1 + 159
	7   JavaNativeFoundation                0x0000000126442d3a +[JNFRunLoop _performCopiedBlock:] + 17
	8   Foundation                          0x00007fff99a6fd00 __NSThreadPerformPerform + 293
	9   CoreFoundation                      0x00007fff95493a01 __CFRUNLOOP_IS_CALLING_OUT_TO_A_SOURCE0_PERFORM_FUNCTION__ + 17
	10  CoreFoundation                      0x00007fff95485c5c __CFRunLoopDoSources0 + 476
	11  CoreFoundation                      0x00007fff954851bf __CFRunLoopRun + 927
	12  CoreFoundation                      0x00007fff95484bd8 CFRunLoopRunSpecific + 296
	13  HIToolbox                           0x00007fff98c4656f RunCurrentEventLoopInMode + 235
	14  HIToolbox                           0x00007fff98c461ee ReceiveNextEventCommon + 179
	15  HIToolbox                           0x00007fff98c4612b _BlockUntilNextEventMatchingListInModeWithFilter + 71
	16  AppKit                              0x00007fff9c0608ab _DPSNextEvent + 978
	17  AppKit                              0x00007fff9c05fe58 -[NSApplication nextEventMatchingMask:untilDate:inMode:dequeue:] + 346
	18  AppKit                              0x00007fff9c055af3 -[NSApplication run] + 594
	19  libglass.dylib                      0x0000000128dea5ec -[GlassApplication runLoop:] + 1836
	20  Foundation                          0x00007fff99a6fd00 __NSThreadPerformPerform + 293
	21  CoreFoundation                      0x00007fff95493a01 __CFRUNLOOP_IS_CALLING_OUT_TO_A_SOURCE0_PERFORM_FUNCTION__ + 17
	22  CoreFoundation                      0x00007fff95485b8d __CFRunLoopDoSources0 + 269
	23  CoreFoundation                      0x00007fff954851bf __CFRunLoopRun + 927
	24  CoreFoundation                      0x00007fff95484bd8 CFRunLoopRunSpecific + 296
	25  java                                0x000000010529b463 CreateExecutionEnvironment + 871
	26  java                                0x00000001052971ac JLI_Launch + 1952
	27  java                                0x000000010529d4c0 main + 101
	28  java                                0x0000000105296a04 start + 52
	29  ???                                 0x000000000000000c 0x0 + 12
)
7559 [pool-3-thread-2] DEBUG com.kodcu.service.FileWatchService - Watchservice started for: /Users/markus 
7802 [pool-3-thread-3] DEBUG c.k.service.ui.FileBrowseService - Filesystem Tree relisted for /Users/markus 
1   0x13382e2fc JSC::throwStackOverflowError(JSC::ExecState*)
2   0x1336e31f0 JSC::Interpreter::executeCall(JSC::ExecState*, JSC::JSObject*, JSC::CallType, JSC::CallData const&, JSC::JSValue, JSC::ArgList const&)
3   0x13381313e JSC::call(JSC::ExecState*, JSC::JSValue, JSC::CallType, JSC::CallData const&, JSC::JSValue, JSC::ArgList const&)
4   0x1338816c0 JSC::JSObject::defaultValue(JSC::JSObject const*, JSC::ExecState*, JSC::PreferredPrimitiveType)
5   0x13388610b JSC::JSObject::toString(JSC::ExecState*) const
6   0x133172476 Java_com_sun_webkit_dom_JSObject_toStringImpl
7   0x1074a79d4
#
# A fatal error has been detected by the Java Runtime Environment:
#
#  SIGSEGV (0xb) at pc=0x0000000133535b1e, pid=32264, tid=94323
#
# JRE version: Java(TM) SE Runtime Environment (8.0_66-b17) (build 1.8.0_66-b17)
# Java VM: Java HotSpot(TM) 64-Bit Server VM (25.66-b17 mixed mode bsd-amd64 compressed oops)
# Problematic frame:
# C  [libjfxwebkit.dylib+0xe2db1e]  WTFCrash+0x3e
#
# Failed to write core dump. Core dumps have been disabled. To enable core dumping, try "ulimit -c unlimited" before starting Java again
#
# An error report file with more information is saved as:
# /Users/markus/src/AsciidocFX/hs_err_pid32264.log
#
# If you would like to submit a bug report, please visit:
#   http://bugreport.java.com/bugreport/crash.jsp
# The crash happened outside the Java Virtual Machine in native code.
# See problematic frame for where to report the bug.
#
[1]    32264 abort      target/appassembler/bin/asciidocfx.sh
```

Can I provide more helpful information?

thank you

**Comment by madmas** [comment]
Just realized taht this seems to be a duplicate of #150 and maybe #180. Sorry for the duplicate. Leavin it open for the moment in case I can collect more information...

**Comment by rahmanusta** [comment]
Thank you for report @madmas , I tried your case and I'm getting an exception while typing second : but not caused to JVM crash in Windows. I will investigate the reason tomorrow.

**Comment by rahmanusta** [comment]
I filed a bug in asciidoctor.js https://github.com/asciidoctor/asciidoctor.js

**Comment by rahmanusta** [comment]
Could you please try version 1.4.5 ? https://github.com/asciidocfx/AsciidocFX/releases
